File: problemas/iniciante/1064/1064.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    double numero, media;
    int qtd;

    media = 0;
    qtd = 0;

    for (int i = 0; i < 6; ++i)
    {
        cin >> numero;

        if (numero > 0)
        {
            qtd += 1;
            media += numero;
        }
    }

    if (qtd > 0)
    {
        media /= qtd;
    }

    cout << qtd << " valores positivos" << endl;
    cout << setprecision(1) << fixed << media << endl;

    return 0;
}

File: problemas/iniciante/1064/1064.cs
using System;

class URI {
    static void Main(string[] args) {
        int contador = 0;
        double numero, media = 0.0;
        
        for(int i = 0; i < 6; ++i){
            numero = double.Parse(Console.ReadLine());
            
            if(numero > 0.0){
                ++contador;
                media += numero;
            }
        }
        
        if(contador > 0){
            media /= contador;
        }
        
        Console.WriteLine($"{contador} valores positivos");
        Console.WriteLine($"{media:0.0}");
    }
}